gffquant/counters/count_manager.py

```
@@ -1,5 +1,7 @@
 """count_manager"""
 
+import csv
```

Flake8 (7.3.0)

[error] 'csv' imported but unused

(F401)